device/mellanox/x86_64-mlnx_msn4410-r0/plugins/eeprom.py

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/plugins/eeprom.py
```

Comment:
are you sure? i prefer in any case it will be using the sn4700 even for visuality

Reply:
Generally, all the plugins are common, except the PSU plugin in 2010 and 2100, because of non-replaceable PSU installed on them.